packages/react/src/React.js

```diff
@@ -141,18 +140,15 @@ if (enableScopeAPI) {
 // Make sure that stable builds for open source
 // don't modify the React object to avoid deopts.
 // Also let's not expose their names in stable builds.
-
-if (enableJSXTransformAPI) {
-  if (__DEV__) {
-    React.jsxDEV = jsxWithValidation;
-    React.jsx = jsxWithValidationDynamic;
-    React.jsxs = jsxWithValidationStatic;
-  } else {
-    React.jsx = jsx;
-    // we may want to special case jsxs internally to take advantage of static children.
-    // for now we can ship identical prod functions
-    React.jsxs = jsx;
-  }
+if (__DEV__) {
+  React.jsxDEV = jsxWithValidation;
+  React.jsx = jsxWithValidationDynamic;
```

Comment:
Would you consider moving `.jsx` and ` .jsxs` up to Line 105? 
```jsx
  jsx: __DEV__ ? jsxWithValidationDynamic : jsx,
  jsxs: __DEV__ ? jsxWithValidationStatic : jsx,

```
and leave `React.jsxDEV = jsxWithValidation;` as it is here in this `__DEV__` check

Just for consistency with createElement.